README.md

```diff
@@ -58,69 +58,44 @@ The following commands always act on the `MODULE` file of the current module (ac
-
-Examples:
-* `dbt dep update` will update all dependency entries of the current module
-* `dbt dep update moduleA moduleB` will update the dependency entries for dependencies to modules `moduleA` and `moduleB` for the current module
-* `dbt dep update --all` will update all dependency entries of all modules in the workspace
-* `dbt dep update --all moduleC` will update the dependency entries for dependencies to module `moduleC` for all modules in the workspace
-
 ### Module initialization
 
-If a module has a `SETUP.go` file in its root directory, DBT will run the `SETUP.go` file when the module is initially cloned or downloaded. This mechanism can be be used to initialize modules (e.g. install git hooks).
+If a module has a `SETUP.go` file in its root directory, DBT will run the `SETUP.go` whenever a new snapshot of the module is checked out. This mechanism can be be used to initialize modules (e.g. install git hooks). The `SETUP.go` scripts should thus be written in an idempotent way. DBT enforces a 10 second time limit on `SETUP.go` scripts.
 
 ### The sync command
 
-The `dbt sync [--master]` command recursively clones, downloads and updates modules to satisfy the dependencies declared in the `MODULE` files starting from the top-level module. All dependencies to a module must resolve to the same version hash. If DBT encounters any conflicting version hashes for the same dependency across `MODULE` files, the sync operation fails.
-
-If the `--master` flag is specified, DBT will ignore all versions specified in `MODULE` files and use `master` as the version for all dependencies.
-
-### The fetch command
-
-`dbt fetch` will run `git fetch` in all Git modules in the workspace. Modules that have uncommited changes will be skipped. This is mostly useful in combination with the `git dep update` command.
-
-### The status command
+The `dbt sync [--update] [--latest] [--ignore-errors]` command recursively clones, downloads and updates modules to satisfy the dependencies declared in the `MODULE` files starting from the top-level module. All dependencies to a module must resolve to the same version. If DBT encounters any conflicting version hashes for the same dependency across `MODULE` files, the sync operation fails. If the `--ignore-errors` flag is used, errors related to mismatcing dependency URLs or versions will be ignored.
 
-The `dbt status` command prints a summary of all currently available modules and their current versions, and reports any unsatisied dependencies. The command will never perform any changes on any module. 
+If the `--update` flag is used, DBT will ignore all previously resolved dependency hashes.
+If the `--latest` flag is used, DBT will ignore the versions specified in `MODULE` files and use `origin/master` as the version for all dependencies.
+`--latest` imples `--update`.
```

Comment:
`--latest` will always use the `origin/master` branch and completely ignore what is specified in the `MODULE` file. The idea is that it may be used by CI to make sure all master branches work together (if that is a requirement). If you want the latest updates from the same branch, `--update` will be enough. But I agree that `--latest` is a bad name for this. Will change it to `--master`.